internal/api/lock.go

```diff
@@ -13,8 +13,8 @@ import (
 
 // lockCluster synchronizes access to the given cluster across potentially
 // multiple provisioning servers.
-func lockCluster(c *Context, clusterID string) (*model.Cluster, int, func()) {
-	cluster, err := c.Store.GetCluster(clusterID)
+func lockCluster(c *Context, clusterID string) (*model.ClusterDTO, int, func()) {
```

Comment:
For this function and others where the return type has changed to a DTO object, we need to change the name of the variables at the call sites from e.g. `cluster` to `clusterDTO` to avoid the `cluster.Cluster` stutter and also to reflect the new type in those locations.

Reply:
`go-rename` should make short work of this.